code/go/0chain.net/blobbercore/datastore/postgres.go

```diff
@@ -84,6 +85,11 @@ func (store *postgresStore) Close() {
 }
 
 func (store *postgresStore) CreateTransaction(ctx context.Context) context.Context {
+	//conn := ctx.Value(ContextKeyTransaction)
+	//if conn != nil {
+	//	return ctx
+	//}
```

Comment:
should we remove this section? I see this causing confusions later.

Reply:
for sure